FLT/HIMExperiments/module_topology.lean

```diff
@@ -213,6 +216,111 @@ noncomputable def LinearMap.add (M : Type*) [AddCommGroup M] [Module A M] :
   map_add' _ _ := add_add_add_comm _ _ _ _
   map_smul' _ _ := (DistribSMul.smul_add _ _ _).symm
 
+/-- Basis.repr in the first variable as a linear map. -/
+noncomputable def LinearMap.basis₁ (M : Type*) (ι : Type*) [Finite ι] [AddCommGroup M] [Module A M]
+  (b : Basis ι A M) (i : ι): M →ₗ[A] A where
```

Comment:
```suggestion
    (b : Basis ι A M) (i : ι) : M →ₗ[A] A where
```
Everything before `where` or `:=` needs to be indented 4 spaces if you go beyond 1 line. Also in the next three declarations.